catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -218,13 +218,13 @@ protected Object executeWithSubject() throws Exception {
                         } catch (IOException | CatalogTransformerException e) {
                             transformationFailed = true;
                             LOGGER.debug("Failed to dump metacard {}", metacard.getId(), e);
-                            executorService.shutdownNow();
                         }
-                        printStatus(resultCount.incrementAndGet());
```

Comment:
Just realized I moved this to the wrong place, it should be inside the result loop not out here. Just moved it back in.